app/src/main/java/org/systers/mentorship/view/activities/FeedbackActivity.kt

```diff
@@ -0,0 +1,211 @@
+package org.systers.mentorship.view.activities
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.text.Editable
+import org.systers.mentorship.R
+import android.util.Patterns
+import androidx.core.app.ComponentActivity
+import androidx.core.app.ComponentActivity.ExtraData
+import androidx.core.content.ContextCompat.getSystemService
+import android.text.TextUtils
+import android.text.TextWatcher
+import android.util.Log
+import android.view.View
+import android.widget.ImageButton
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_feedback.*
+
+class FeedbackActivity : AppCompatActivity(), View.OnClickListener {
```

Comment:
```suggestion
class FeedbackActivity : BaseActivity(), View.OnClickListener {
```

Reply:
can you elaborate?

Reply:
We have a `BaseActivity` which extends from `AppCompatActivity`. It has some functionality which is common to all activities eg TAGs, dialogs etc.

Reply:
created this issue https://github.com/anitab-org/mentorship-android/issues/624 for this ^^